tests/api/db/test_runs.py

```diff
@@ -57,6 +57,54 @@ def test_list_runs_name_filter(db: DBInterface, db_session: Session):
     assert len(runs) == 2
 
 
+# running only on sqldb cause filedb is not really a thing anymore, will be removed soon
+@pytest.mark.parametrize(
+    "db,db_session", [(dbs[0], dbs[0])], indirect=["db", "db_session"]
+)
+def test_list_distinct_runs_uids(db: DBInterface, db_session: Session):
+    project_name = "project"
+    # create 3 runs with same uid but different iterations
+    for i in range(3):
+        _create_new_run(db, db_session, project=project_name, iteration=i)
```

Comment:
```suggestion
        _create_new_run(db, db_session, project=project_name, iteration=i, uid="test-uid1")
```

Why not create with a given `uid` and verify that you get the correct `uid` back?